src/test/javascript/jsdoc_test.ts

```diff
@@ -76,7 +76,7 @@ suite('jsdoc', function() {
 
     test('preserves indentation for the body', function() {
       const parsed = jsdoc.parseJsdoc('*     The desc.\n*     thing');
-      assert.deepEqual(parsed.description, '    The desc.\n    thing');
+      assert.deepEqual(parsed.description, 'The desc.\n    thing');
```

Comment:
Yeah, I couldn't think of a reason to keep it, but you're right, we don't want to be tracking down weird errors right now. Chesterton's Fence beats Cargo Cult for now. Added a todo to look into it.

Reverted.